### ISSUE TITLE
isValidAgainst doesn't override custom message

### ISSUE THREAD

**Issue by Wolf05632** [xmlunit/xmlunit#225]
isValidAgainst method in assertj3.XmlAssert module doesn't override custom error message:

```
XmlAssert
.assertThat(xmlSource)
.withFailMessage("Alarm alarm!")
.isValidAgainst(xsdSource);
```

expected:

```
java.lang.AssertionError: Alarm alarm!
Expecting: ...
```

fact:

```
java.lang.AssertionError:
Expecting: ...
```

**Comment by bodewig** [comment]
I must admit that I'm nowhere close to being an AssertJ expert, so if you know how to fix it, I'd be grateful for a PR. If I need to figure things out, it is gong to take a bit longer, I'm afraid.

**Comment by bodewig** [comment]
OK, it took quite a bit more effort than I thought it would take, but I think it all assertions now honor overridden error messages.

**Comment by bodewig** [comment]
Sorry, @krystiankaluzny I didn't see you started working on it as well. Maybe we can merge your changes with mine in any way?

**Comment by krystiankaluzny** [comment]
Hi @bodewig I didn't notice that here are notifications about my commits :) 
I don't see a problem to merge our changes. I will try soon to apply my changes to the current main branch

**Comment by bodewig** [comment]
@krystiankaluzny I only saw the notifications here when I came back to comment on the issue :-)

I agree it should be possible to merge your approach of extracting the configuration and inheriting the info property with my changes - I liked your approach better than mine. Actually I've not been happy with duplicating the code that copied the info properties to three places myself.

For the assetj3 module I toyed with the idea of getting rid of our own `allSatisfy` method in `MultipleNodeAssert` in favor of the built-in one, but this would result in the assertion error messages changing (we'd lose the custom navigationDescription) so I backed off.
